tests/integration/test_get_deposits.py

```diff
@@ -423,3 +432,212 @@ def test_get_deposit_with_permissions_json_serializer_returns_serialized_permiss
             }
         }
     }
+
+
+def test_get_deposit_with_form_json_serializer_1(
+        client, db, auth_headers_for_example_user, example_user,
+        create_deposit, create_schema, file_tar):
+
+    # create schema
+    ###############
+    create_schema('test-schema', experiment='CMS',
+                  deposit_schema={
+                      'title': 'deposit-test-schema', 'type': 'object',
+                      'properties': {
+                            'title': {'type': 'string'},
+                            'date': {'type': 'string'}
+                      }},
+                  deposit_options={
+                      'title': 'ui-test-schema', 'type': 'object',
+                      'properties': {
+                          'title': {'type': 'string'},
+                          'field': {'type': 'string'}
+                      }})
+    deposit = create_deposit(example_user, 'test-schema', experiment='CMS')
+
+    # create file
+    #############
+    # deposit.save_file(b'Hello!', 'README.md', 6)
+
+    # create webhooks
+    #################
+    repo = GitRepository(external_id=12345,
+                         host='github.com',
+                         owner='owner',
+                         name='repository')
+    webhook = GitWebhook(event_type='push',
+                         repo_id=repo.id,
```

Comment:
one small thing, not really test-related
do you know that you're assigning nothing to repo_id here? ^^
to check what is happening, put a breakpoint here and take a look at what your repo object is, and figure out why it doesn't have an id